test/components/DayPickerRangeController_spec.jsx

```diff
@@ -796,6 +796,140 @@ describe('DayPickerRangeController', () => {
         });
       });
 
+      describe('blocked', () => {
+        describe.skip('focusedInput did not change', () => {
```

Comment:
So you probably have to reset the stub after the initial shallow render. The first mount is going to call this method a bunch of times! You want to isolate it to make sure that you're only checking how many times `componentWillReceiveProps` is calling it!